### DISCUSSION TITLE
Add an option to the `biome ci` command to not output the "The file contains diagnostics that needs to be addressed" message

### DISCUSSION THREAD

**Discussion by mongolyy** [biomejs/biome#1763] [Ideas]
## Background

I am using `biome ci` to develop tools for use in CI.

https://github.com/marketplace/actions/run-biome-with-reviewdog

## Description

The message "The file contains diagnostics that need to be addressed" and the detailed message about the diagnostics that need to be addressed are output as separate errors.

The "The file contains diagnostics that need to be addressed" has a duplicate role, so my CI tool filters the output.
It would be useful to have an option to not output the "The file contains diagnostics that need to be addressed" message in the first place, so I don't have to do this filtering!

**Comment by ematipico** [comment]
Are you reading `stderr` or `stdout`?

**Reply by mongolyy** [reply]
Thanks for the response.
I'm reading stderr.

**Comment by Conaclos** [comment]
We could remove this error? It is an unneeded diagnostic?

**Comment by ematipico** [comment]
Implemented in https://github.com/biomejs/biome/pull/1808
